plugins/policy-recommendation/policy_recommendation_job.py

```diff
@@ -0,0 +1,849 @@
+#!/usr/bin/python3
+
+# Copyright 2022 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import getopt
+import json
+import os
+import random
+import string
+import sys
+import uuid
+
+import kubernetes.client
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import udf
+from pyspark.sql.types import StringType
+from urllib.parse import urlparse
+
+import antrea_crd
+from policy_recommendation_utils import *
+
+# Column names of flow record table in Clickhouse database used in recommendation job
+FLOW_TABLE_COLUMNS = [
+    'sourcePodNamespace',
+    'sourcePodLabels',
+    'destinationIP',
+    'destinationPodNamespace',
+    'destinationPodLabels',
+    'destinationServicePortName',
+    'destinationTransportPort',
+    'protocolIdentifier',
+]
+
+NAMESPACE_ALLOW_LIST = [
+    'kube-system',
+    'flow-aggregator',
+    'flow-visibility'
+]
+
+ROW_DELIMITER = "#"
+PEER_DELIMITER = "|"
+DEFAULT_POLICY_PRIORITY = 5
+
+MEANINGLESS_LABELS = [
+    "pod-template-hash",
+    "controller-revision-hash",
+    "pod-template-generation"
+]
+
+spark = SparkSession.builder.getOrCreate()
+spark.sparkContext.setLogLevel("info")
```

Comment:
Should we make the log level configrable? It may help for live debug purpose.

Reply:
Thank Salvatore help me answer this question. 
Yes, I hard-coded log level to info because I only added logs at info, warning, and error level. Also tried changing to "debug" mode and I could see lots of debug logs automatically generated by Spark.